src/core_plugins/kibana/public/management/sections/objects/change_index_modal.js

```diff
@@ -0,0 +1,207 @@
+                return (
+                  <option key={i} value={index.id}>
+                    {index.get('title')}
+                  </option>
+                );
+              })}
+            </select>
+          </KuiTableRowCell>
+        </KuiTableRow>
+      );
+    });
+
+    return (
+      <KuiModalOverlay>
+        <KuiModal
+          data-tests-subj="managementChangeIndexModal"
+          aria-label="Index does not exist"
+          className="managementChangeIndexModal"
+          onKeyDown={this.onKeyDown}
+        >
+          <KuiModalHeader>
+            <KuiModalHeaderTitle>
+              Index Pattern Conflicts
+            </KuiModalHeaderTitle>
+          </KuiModalHeader>
+          <KuiModalBody>
+            <KuiModalBodyText>
+              <p>The following saved objects use index patterns that do not exist.</p>
+            </KuiModalBodyText>
+            <KuiControlledTable>
+              <KuiToolBar>
+                <KuiToolBarSection>
+                  { totalIndexPatterns > perPage
+                    ?
+                      <KuiPager
+                        startNumber={page + 1}
+                        hasPreviousPage={page >= 1}
+                        hasNextPage={page < totalIndexPatterns}
+                        endNumber={Math.min(totalIndexPatterns, page + perPage)}
+                        totalItems={totalIndexPatterns}
+                        onNextPage={() => this.setState({ page: page + 1 })}
+                        onPreviousPage={() => this.setState({ page: page - 1 })}
+                      />
+
+                    : null
+                  }
+                </KuiToolBarSection>
+              </KuiToolBar>
+              <KuiTable className="kuiVerticalRhythm">
+                <KuiTableHeader>
+                  <KuiTableHeaderCell width="300">
+                    ID
+                  </KuiTableHeaderCell>
+                  <KuiTableHeaderCell width="50">
+                    Count
+                  </KuiTableHeaderCell>
+                  <KuiTableHeaderCell>
+                    Sample of Affected Objects
+                  </KuiTableHeaderCell>
+                  <KuiTableHeaderCell width="200">
+                    New Index
+                  </KuiTableHeaderCell>
+                </KuiTableHeader>
+                <KuiTableBody>
+                  {rows}
+                </KuiTableBody>
+              </KuiTable>
+            </KuiControlledTable>
+          </KuiModalBody>
+
+          <KuiModalFooter>
+            <KuiButton
+              buttonType="hollow"
+              data-test-subj="cloneCancelButton"
+              onClick={this.props.onClose}
+            >
+              Cancel
+            </KuiButton>
+            <KuiButton
+              buttonType="primary"
+              data-test-subj="cloneConfirmButton"
+              onClick={this.changeIndex}
+            >
+              Confirm All Changes
```

Comment:
Can we sentence case this? "Confirm all changes". I think this is how we should case our button text but I'm double-checking with our copywriters.